tools/editor/plugins/resource_preloader_editor_plugin.cpp

```diff
@@ -60,36 +60,40 @@ void ResourcePreloaderEditor::_notification(int p_what) {
 	}
 }
 
-void ResourcePreloaderEditor::_file_load_request(const String& p_path) {
+void ResourcePreloaderEditor::_files_load_request(const Vector<String>& p_path) {
```

Comment:
Or maybe keep the old ``_file_load_request`` name as use the same kind of definition as for the two other ``_file_load_request`` functions: https://github.com/godotengine/godot/search?utf8=%E2%9C%93&q=_file_load_request

- ``SampleLibraryEditor::void _file_load_request(const DVector<String>& p_path)``
- ``SpriteFramesEditor::_file_load_request(const DVector<String>& p_path)``